packages/webapp/pages/_app.tsx

```diff
@@ -199,17 +201,17 @@ function InternalApp({ Component, pageProps, router }: AppProps): ReactElement {
 }
 
 export default function App(props: AppProps): ReactElement {
-  const [queryClient] = useState(
-    () => new QueryClient(defaultQueryClientConfig),
-  );
   const version = useWebappVersion();
   const deviceId = useDeviceId();
   useError();
   useManualScrollRestoration();
 
   return (
     <ProgressiveEnhancementContextProvider>
-      <QueryClientProvider client={queryClient}>
+      <PersistQueryClientProvider
+        client={persistedQueryClient}
+        persistOptions={persistedQueryClientOptions}
+      >
```

Comment:
This will then underlying resolve the regular one as well?

Reply:
The persistedQueryClient is just the regular queryClient with some additional options, and the provider has some additional utilities, so everything that is not related to persisting should be working as it always did